features/step_definitions/browser_steps.rb

```diff
@@ -1,5 +1,16 @@
-Then(/^I see a new (.+?) pull request for the "(.+?)" branch in the "(.+?)" repo in my browser$/) do |domain, branch, repo|
-  expect(@last_run_result.out.strip).to eql "#{@tool} called with: #{pull_request_url domain, branch, repo}"
+Then(/^I see a new (.+?) pull request for the "([^"]+)" branch in the "(.+?)" repo in my browser$/) do |domain, branch, repo|
+  expect(@last_run_result.out.strip).to eql "#{@tool} called with: #{pull_request_url domain: domain,
+                                                                                      branch: branch,
+                                                                                      parent_branch: 'main',
+                                                                                      repo: repo}"
```

Comment:
I think this would be cleaner as
```ruby
url = pull_request_url domain: domain, branch: branch, parent_branch: 'main', repo: repo
expect(@last_run_result.out.strip).to eql "#{@tool} called with: #{url}"
```